.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: ruby
+cache: bundler
```

Comment:
Faster test runs because we don't have to install the gems every time (unless something actually changes!)